product_tier_validation/models/product_template.py

```diff
@@ -8,3 +8,34 @@ class ProductTemplate(models.Model):
     _name = "product.template"
     _inherit = ["product.template", "tier.validation"]
     _tier_validation_manual_config = False
+
```

Comment:
@simahawk, I found where `confirmed` is used.
@rousseldenis, Should we add this?
```python
    _state_to = ["sellable"]
```

Reply:
@hailangvn @simahawk Yes indeed. I first did it in my project to confirm flow.

As 'confirmed' state comes from base_tier_validation, we need to override it here